fern/pages/fern-docs/content/customize-api-ref.mdx

```diff
@@ -0,0 +1,190 @@
+---
+title: Customize API Reference layout
+description: Customize your API Reference's naming, ordering, and structure.  
+---
+
+When you include an API (either the Fern or an OpenAPI definition) in your `docs.yml` file, you can customize how the endpoints and sub-packages are displayed in the sidebar navigation. By default, the reference will generate a navigation hierarchy based on the structure of the API spec, but several customizations can be configured.
```

Comment:
I think the parenthetical here makes for choppy reading. "When you include an API in your ..." is more natural.